README.md

```diff
@@ -1 +1,153 @@
-# space-travelers-hub
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Space Travelers Hub</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [Loom Walkthrough](#loom)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Space Travelers Hub] <a name="about-project"></a>
+
+**Space Travelers Hub**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/BB-Simon/">yml for linters</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JS</a></li>
+    <li><a href="https://legacy.reactjs.org/">ReactJS</a></li>
+    <li><a href="https://github.com">GIT/Github</a></li>
+    <li><a href="https://nodejs.org/en/">NodeJS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **Create react app by create-react-app**
+- **Add button component**
+- **Add Mission  component**
+- **Add my-profile component**
+- **Add Rockets components**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- VSCode or any other IDE.
+- node.js(preferable).
+- browser to deploy.
+- GitHub/Git.
+
+### Setup
+
+> Clone this repository to your desired folder:
+
+git clone https://github.com/BB-Simon/space-travelers-hub
+
+> In the project directory, you can run:
+
+to install dependencies:
+#### `npm install`
+
+to run the project:
+#### `npm start`
+
+to build:
+#### `npm build`
+
```

Comment:
- [OPTIONAL] I think it could be good if you also add the command to run tests 😉

Reply:
Thanks for the review @cynthiainga